Configuration/DataProcessing/python/Merge.py

```diff
@@ -52,7 +52,7 @@ def mergeProcess(*inputFiles, **options):
     #//
     if newDQMIO:
         process.source = Source("DQMRootSource")
-        process.add_(Service("DQMStore"))
+        process.add_(Service("DQMStore", forceResetOnBeginLumi = CfgTypes.untracked.bool(True)))
```

Comment:
@schneiml if the feature needs to be off by default, should not this flag be turned to False?

Reply:
No, the changes in HARVESTING behavior are spread over many places and unconditional. This fixes a number of problems with the existing per-lumi plots (at the risk of causing some misbehavior in said plots, as explained in the PR message). 

Conditional is only the change in the RECO step, which turns on per-lumi saving on all plots (that allow it).